classes/migration/install/MetricsMigration.inc.php

```diff
@@ -0,0 +1,239 @@
+<?php
+
+/**
+ * @file classes/migration/install/MetricsMigration.inc.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class MetricsMigration
+ * @brief Describe database table structures.
+ */
+
+namespace APP\migration\install;
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema as Schema;
+
+class MetricsMigration extends \PKP\migration\Migration
+{
+    /**
+     * Run the migrations.
+     * This migration file is used during upgrades. If this schema changes, the upgrade scripts should be reviewed manually before a merging.
```

Comment:
@asmecher told me to do so, so I will let him decide... :-)

Reply:
I was thinking to do a "lazy copy", where we only actually duplicate it when we need to modify the schema. But I see the sense in what Nate is proposing, so yes, please do duplicate the migration so that the `install` and `upgrade` ones are different.